### ISSUE TITLE
Broken Version of Burrito Bison

### ISSUE THREAD

**Issue by PunchmadeDav** [Gams-Offline/Gams#3]
Your version of burrito bison is broken. The game breaks after the first cake wall

**Comment by Trickiy** [comment]
Once the crash happens once it will persist thru reloads thus being in a softlock.

**Comment by mountain658** [comment]
I thought this was fixed, I'll try it out again. Thanks for finding these btw.

**Comment by Trickiy** [comment]
I think it's worth mentioning that when I click on the + in the shop the game will crash.

**Comment by mountain658** [comment]
Yeah, ok, I'll try to find a new version but the ones I've found has the same thing...

**Comment by mountain658** [comment]
Mine works now!!

**Comment by Trickiy** [comment]
I can't test anymore due to finishing highschool and having my chromebook taken but it is nice to see the game gain more functionality.

**Comment by mountain658** [comment]
Aww man you don't have a PC or laptop? Well thank you for sticking with the project lol 

I took forever because of school and stuff too so I get it
